src/app/components/publiclists/teamlist/teamcard/teamcard.component.html

```diff
@@ -1,41 +1,41 @@
 <div class="team-card-container" [class.selected]='isSelected' [class.noexpand] = 'isHost' [class.expand] = '!isHost'>
 <div class="team-card-container-child">
-  <div class="team-content" (click)='selectToggle()'>
-    <i class="fa fa-circle selectstate" *ngIf='!isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-check selectstate" aria-hidden="true" *ngIf='isSelected && (isHost || isOnChallengePage)'></i>
-    <i class="fa fa-users" aria-hidden="true"></i>
-    <div class="team-details">
-      <span class="title toptitle">
-	    Team:
-      </span>
-      <span *ngIf="teamView['team_url']">
-      	<a class="orange-text" target="_blank" href="{{teamView['team_url']}}">{{teamView['team_name']}}</a>
-      </span>
-      <span *ngIf="!teamView['team_url']">
-	    {{teamView['team_name']}}
-      </span>
-      <br>
-      <span class="title">
-	    Created by:
-      </span>
-      <span>
-	    {{teamView['created_by']}}
-      </span>
+    <div class="team-content" (click)='selectToggle()'>
+      <input type="radio"  class="radio-button__input" *ngIf='!isSelected && (isHost || isOnChallengePage)' id="644">
```

Comment:
Please use unique and meaningful ID's.

Reply:
@lunayach I thought it might be good to se the same I'd as used in main repo. But I will change it

Reply:
oh Sorry Sir my mistake i didn't see the ids are generated randomly